javascript/ql/src/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -1235,9 +1247,17 @@ module PartialInvokeNode {
     DataFlow::SourceNode getBoundFunction(DataFlow::Node callback, int boundArgs) { none() }
 
     /**
+     * DEPRECATED. Use the two-argument version of `getBoundReceiver` instead.
```

Comment:
**Two**-argument version? You have introduced a **one**-argument version below: `getBoundReceiver(DataFlow::Node callback) ` (counting the implicit `this` among the arguments in this docstring is confusing IMO)

Reply:
Indeed, changed to `one-argument version`